2023-03-08 16:10:15,402 INFO    MainThread:25920 [wandb_setup.py:_flush():68] Configure stats pid to 25920
2023-03-08 16:10:15,402 INFO    MainThread:25920 [wandb_setup.py:_flush():68] Loading settings from C:\Users\canba\.config\wandb\settings
2023-03-08 16:10:15,403 INFO    MainThread:25920 [wandb_setup.py:_flush():68] Loading settings from C:\Users\canba\OneDrive\Masaüstü\OPFGNN\code\wandb\settings
2023-03-08 16:10:15,403 INFO    MainThread:25920 [wandb_setup.py:_flush():68] Loading settings from environment variables: {'_require_service': 'True'}
2023-03-08 16:10:15,403 INFO    MainThread:25920 [wandb_setup.py:_flush():68] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2023-03-08 16:10:15,403 INFO    MainThread:25920 [wandb_init.py:_log_setup():492] Logging user logs to C:\Users\canba\OneDrive\Masaüstü\OPFGNN\code\wandb\run-20230308_161015-o816f3xi\logs\debug.log
2023-03-08 16:10:15,403 INFO    MainThread:25920 [wandb_init.py:_log_setup():493] Logging internal logs to C:\Users\canba\OneDrive\Masaüstü\OPFGNN\code\wandb\run-20230308_161015-o816f3xi\logs\debug-internal.log
2023-03-08 16:10:15,403 INFO    MainThread:25920 [wandb_init.py:_jupyter_setup():438] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x0000020312A30908>
2023-03-08 16:10:15,403 INFO    MainThread:25920 [wandb_init.py:init():532] calling init triggers
2023-03-08 16:10:15,403 INFO    MainThread:25920 [wandb_init.py:init():539] wandb.init called with sweep_config: {}
config: {'learning_rate': 0.001, 'architecture': 'Homo-GNN', 'dataset': '1-HV-mixed--0-no_sw', 'epochs': 100, 'activation': 'ReLu', 'dropout': 0.0}
2023-03-08 16:10:15,403 INFO    MainThread:25920 [wandb_init.py:init():588] starting backend
2023-03-08 16:10:15,403 INFO    MainThread:25920 [wandb_init.py:init():592] setting up manager
2023-03-08 16:10:15,405 INFO    MainThread:25920 [backend.py:_multiprocessing_setup():108] multiprocessing start_methods=spawn, using: spawn
2023-03-08 16:10:15,407 INFO    MainThread:25920 [wandb_init.py:init():599] backend started and connected
2023-03-08 16:10:15,416 INFO    MainThread:25920 [wandb_run.py:_label_probe_notebook():1203] probe notebook
2023-03-08 16:10:15,416 INFO    MainThread:25920 [wandb_run.py:_label_probe_notebook():1213] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2023-03-08 16:10:15,416 INFO    MainThread:25920 [wandb_init.py:init():687] updated telemetry
2023-03-08 16:10:15,461 INFO    MainThread:25920 [wandb_init.py:init():727] communicating run to backend with 60.0 second timeout
2023-03-08 16:10:16,227 INFO    MainThread:25920 [wandb_run.py:_on_init():2134] communicating current version
2023-03-08 16:10:16,491 INFO    MainThread:25920 [wandb_run.py:_on_init():2143] got version response upgrade_message: "wandb version 0.13.11 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2023-03-08 16:10:16,492 INFO    MainThread:25920 [wandb_init.py:init():775] starting run threads in backend
2023-03-08 16:10:20,391 INFO    MainThread:25920 [wandb_run.py:_console_start():2114] atexit reg
2023-03-08 16:10:20,391 INFO    MainThread:25920 [wandb_run.py:_redirect():1969] redirect: SettingsConsole.WRAP_RAW
2023-03-08 16:10:20,391 INFO    MainThread:25920 [wandb_run.py:_redirect():2034] Wrapping output streams.
2023-03-08 16:10:20,391 INFO    MainThread:25920 [wandb_run.py:_redirect():2059] Redirects installed.
2023-03-08 16:10:20,392 INFO    MainThread:25920 [wandb_init.py:init():817] run started, returning control to user process
2023-03-08 16:10:20,394 INFO    MainThread:25920 [jupyter.py:save_ipynb():379] not saving jupyter notebook
2023-03-08 16:10:20,394 INFO    MainThread:25920 [wandb_init.py:_pause_backend():403] pausing backend
2023-03-08 16:10:56,220 INFO    MainThread:25920 [wandb_init.py:_resume_backend():408] resuming backend
2023-03-08 16:10:57,361 INFO    MainThread:25920 [jupyter.py:save_ipynb():379] not saving jupyter notebook
2023-03-08 16:10:57,361 INFO    MainThread:25920 [wandb_init.py:_pause_backend():403] pausing backend
2023-03-08 16:14:32,886 WARNING MsgRouterThr:25920 [router.py:message_loop():77] message_loop has been closed
